### ISSUE TITLE
Getter and Setter for model in DataTablesEditor.php

### ISSUE THREAD

**Issue by schonhoff** [yajra/laravel-datatables-editor#24]
### Summary of problem or feature request

It would be very useful for the DataTablesEditor abstract class to have getter and setter for the protected model attribute. Sometimes I need to remind me to fill the protected model attribute in the extended class. But if I could set/get the model outside of the class would be very useful for a laravel controller use-case.

### Code snippet of problem

```php
abstract class DataTablesEditor
{
    public function getModel() {
        return $this->model;
    }

    public function setModel($model) {
        $this->model = $model;
    }
```
So I can call in the store function in a Laravel controller:
```php
        $editor->setModel(Model::class);
        return $editor->process(request());
```

### System details

- Operating System: Windows 7
- PHP Version: 7.2.14
- Laravel Version: 5.7
- Laravel-DataTables Version: 1.6.1

**Comment by yajra** [comment]
@schonhoff Sorry for late response, the idea looks good for me. Would you mind sending a PR? Thanks!

**Comment by yajra** [comment]
I went ahead and created the PR. Thanks for the idea!

**Comment by yajra** [comment]
Released on v1.8.0 🚀
